README.md

```diff
@@ -37,7 +37,7 @@ Property | Type | Description
 `contact_types.name` | string | The name of the contact_type as it [appears in the app's base_settings.json](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/hierarchy/)
 `contact_types.friendly` | string | Friendly name of the contact type
 `contact_types.contact_type` | string | The contact_type of the primary contact. [As defined in base_settings.json](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/hierarchy/)
-`contact_types.user_role` | string | The [role](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/user-roles/) of the user which is created
+`contact_types.user_role` | string[] | The [role](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/user-roles/) to assign to the created user. When multiple roles are available, this property acts as a list of allowed roles against which the specified role(s) should be checked.
```

Comment:
```suggestion
`contact_types.user_role` | string[] | A list of allowed [user roles](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/user-roles/). If only one is provided, it will be assigned automatically.
```